index.js

```diff
@@ -3,11 +3,23 @@ const path = require('path');
 const Conf = require('./lib/conf');
 const _defaults = require('./lib/defaults');
 
-// https://github.com/npm/npm/blob/latest/lib/config/core.js#L101-L200
+// https://github.com/npm/cli/blob/latest/lib/config/core.js#L101-L200
 module.exports = (opts, types, defaults) => {
 	const conf = new Conf(Object.assign({}, _defaults.defaults, defaults), types);
 
 	conf.add(Object.assign({}, opts), 'cli');
+
+	const paths = require.resolve.paths('npm');
```

Comment:
I suggest making it backward compatible. Wrap this code in a `if (require.resolve) {`. In that case, it will work on older versions of Node as well

Reply:
Added